lib/cocoapods/target/framework_paths.rb

```diff
@@ -9,14 +9,19 @@ class FrameworkPaths
       #
       attr_reader :dsym_path
 
-      def initialize(source_path, dsym_path = nil)
+      # @return [String, Nil] the bcsymbolmap files path array, if one exists
```

Comment:
is this an Array now? `[Array<String>, Nil]`

Reply:
Yes